examples/simulation/rutherford/src/FairRutherford.h

```diff
@@ -67,30 +67,30 @@ class FairRutherford : public FairDetector
 
     //    virtual void   CopyClones( TClonesArray* cl1,  TClonesArray* cl2 ,
     //                               Int_t offset) {;}
-    virtual void SetSpecialPhysicsCuts() {}
-    virtual void EndOfEvent();
-    virtual void FinishPrimary() {}
-    virtual void FinishRun() {}
-    virtual void BeginPrimary() {}
-    virtual void PostTrack() {}
-    virtual void PreTrack() {}
-    virtual void BeginEvent() {}
+    void SetSpecialPhysicsCuts() override {}
+    void EndOfEvent() override;
+    void FinishPrimary() override {}
+    void FinishRun() override {}
+    void BeginPrimary() override {}
+    void PostTrack() override {}
+    void PreTrack() override {}
+    void BeginEvent() override {}
 
-    virtual FairModule* CloneModule() const;
+    FairModule* CloneModule() const override;
 
   private:
     static FairRutherfordGeo* fgGeo;   //!
 
     /** Track information to be stored until the track leaves the
     active volume.
     */
-    Int_t fTrackID;        //!  track index
-    Int_t fVolumeID;       //!  volume id
-    TLorentzVector fPos;   //!  position at entrance
-    TLorentzVector fMom;   //!  momentum at entrance
-    Double32_t fTime;      //!  time
-    Double32_t fLength;    //!  length
-    Double32_t fELoss;     //!  energy loss
+    Int_t fTrackID = -1;        //!  track index
```

Comment:
What about using consistently {} for initialisation for all members?

Reply:
Good point!

I was considering to use `=` for base types and `{}` for classes. But maybe we should really just use `{}` all around?

Reply:
Okay, changed over. Should be resolved for this one.